### PR TITLE
Python

### PR DESCRIPTION
**Pull request by beccaelenzil**
Activity converted to a python scaffold for use with the Problem Set and Activity in the Intro to Tests section of the core curriculum.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+## Learning Goals 
+- Practice identifying test cases
+- Practice writing tests with pytest
+- Practice the Test Driven Development (TDD) programming workflow
 
 ## About BlackJack
 
+In this exercise we will write a method and set of tests in TDD fashion which calculates a hand's BlackJack score.
+
 In the card BlackJack each card has a value.
```
Comment (CheezItMan) [inline README.md]:
```suggestion
In the card game BlackJack each card has a value.
```

File: README.md [hunk 2]
```diff
@@ -15,22 +20,32 @@ A hand, an array of Card values, must be between 2 and 5 items inclusive.
...
-3.  How would you **test** these cases?
+## Classroom activity
 
-When you finish talk with another pair and walk through your answers.
+### Part 2:  Identifying Edge & Nominal Cases
 
-## Part 2:  Calculating a Score
+Your classroom instructor will provide instructions on how to complete the following steps.
+1. Review the test cases you identified in Part 1
+1. Determine how you would test these cases
```
Comment (CheezItMan) [inline README.md]:
```suggestion
With your classroom instructor you will:

1. Review the test cases you identified in Part 1
1. Determine how you would test these cases
```

**Review by CheezItMan — APPROVED** [review]
I like these modifications to the Tdd Exercise.  

One caution, a lot of people don't know blackjack, and the rules take a bit to learn.

File: README.md [hunk 3]
```diff
@@ -15,22 +20,32 @@ A hand, an array of Card values, must be between 2 and 5 items inclusive.
...
-1.  What would be two nominal cases?
-2.  What edge cases can you determine?  Name at least 3.
-3.  How would you **test** these cases?
+## Classroom activity
 
-When you finish talk with another pair and walk through your answers.
+### Part 2:  Identifying Edge & Nominal Cases
 
-## Part 2:  Calculating a Score
+Your classroom instructor will provide instructions on how to complete the following steps.
```
Comment (beccaelenzil) [inline README.md]:
```suggestion
In class, you will:
```
